src/templates/components/container.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default" ng-controller="formioContainerComponent">
+  <div class="panel-heading" ng-init="component.hideLabel = true">
+    <h3 class="panel-title">{{ component.label}}</h3>
+  </div>
+  <div class="panel-body">
+    <div ng-repeat="containerData in data[component.key] track by $index">
+      <div ng-repeat="component in component.components">
```

Comment:
I don't see how this would work.  You are iterating over all the keys within the data object, only to then iterate over all the components within the container for every key within the object. If you have more than one item saved in the value, it would essentially duplicate all the fields over and over. Please rework this.